src/check.rs

```diff
@@ -33,15 +35,27 @@ fn check_url(url: &Url) -> bool {
 }
 
 /// Check a URL with the "file" scheme for availability. Returns `false` if it is unavailable.
-fn check_file_url(url: &Url) -> bool {
+fn check_file_url(url: &Url, base_dir: &Path, file: &Path, args: &MainArgs) -> bool {
     let path = url.to_file_path().unwrap();
-    
-    if path.exists() {
-        debug!("Linked file at path {} does exist.", path.display());
-        true
+
+    if  args.flag_debug {
+      if path.exists() {
+          debug!("{}: Linked file {} exists.", file.display(), path.display());
+          true
+      } else {
+          error!("{}: Linked file {} does not exist!", file.display(), path.display());
+          false
+      }
     } else {
-        error!("Linked file at path {} does not exist!", path.display());
-        false
+      let shortpath = path.strip_prefix(base_dir).unwrap_or_else(|_| &path);
+      let shortfile = file.strip_prefix(base_dir).unwrap_or_else(|_| &file);
+      if path.exists() {
+          debug!("{}: Linked file {} exists.", shortfile.display(), shortpath.display());
+          true
+      } else {
+          error!("{}: Linked file {} does not exist!", shortfile.display(), shortpath.display());
+          false
+      }
```

Comment:
Rather than duplicating this, can you lift the `short*` variables out of the if? Something like
```rust
let shortpath = if !args.flag_debug {
  file = file.strip_prefix(base_dir).unwrap_or(&file);
  path.strip_prefix(base_dir).unwrap_or(&path)
} else {
  path
};
```
Then you wouldn't need to write the `path.exists()` if block twice.